docs/UserGuide.md

```diff
@@ -284,11 +299,6 @@ You can also quickly narrow down your choices with the randomizer feature._
 `-h`: displays help message (specific to findCuisine)
 <br>
 
-Note:
-* `PRICE` can be either `$`, `$$`, or `$$$`.
-* Eateries matching at least one price will be returned.
-  e.g. `findPrice $ $$` will return eateries that have either of the prices.
-
```

Comment:
I'm not sure we should remove this part, since price specifically should only accept these 3 values.

Maybe can include a toolbox of some sort or include a note next to the parameters section or something?